access-control-app/project.clj

```diff
@@ -28,18 +28,10 @@
 (defproject nasa-cmr/cmr-access-control-app "0.1.0-SNAPSHOT"
   :description "Implements the CMR access control application."
   :url "https://github.com/nasa/Common-Metadata-Repository/tree/master/access-control-app"
-  :exclusions [[cheshire]
-               [clj-time]
-               [com.fasterxml.jackson.core/jackson-core]
-               [commons-codec/commons-codec]
-               [commons-io]
-               [org.clojure/tools.reader]
-               [ring/ring-codec]]
-  :dependencies ~(concat '[[cheshire "5.10.0"]
+  :exclusions []
```

Comment:
why did you do this?

Reply:
because top-level exclusions affect top-level dependencies, it appeared while I was working on this that the dependencies also listed in the exclusions list just wouldn't be pulled in at all

Reply:
I will remove the empty list